### DISCUSSION TITLE
the trait bound `(diesel::sql_types::BigInt, diesel::sql_types::Text, diesel::sql_types::Numeric, diesel::sql_types::Nullable<diesel::sql_types::Text>): load_dsl::private::CompatibleType<model::CostItems, Pg>` is not satisfied

### DISCUSSION THREAD

**Discussion by cchee** [diesel-rs/diesel#3555] [Q&A]
error[E0277]: the trait bound `(diesel::sql_types::BigInt, diesel::sql_types::Text, diesel::sql_types::Numeric, diesel::sql_types::Nullable<diesel::sql_types::Text>): load_dsl::private::CompatibleType<model::CostItems, Pg>` is not satisfied
    --> src/cost_items/model.rs:39:62
     |
39   |         let cost_items = cost_items::table.load::<CostItems>(&mut conn)?;
     |                                            ----              ^^^^^^^^^ the trait `load_dsl::private::CompatibleType<model::CostItems, Pg>` is not implemented for `(diesel::sql_types::BigInt, diesel::sql_types::Text, diesel::sql_types::Numeric, diesel::sql_types::Nullable<diesel::sql_types::Text>)`
     |                                            |
     |                                            required by a bound introduced by this call
     |
     = help: the following other types implement trait `load_dsl::private::CompatibleType<U, DB>`:
               (ST0, ST1)
               (ST0, ST1, ST2)
               (ST0, ST1, ST2, ST3)
               (ST0, ST1, ST2, ST3, ST4)
               (ST0, ST1, ST2, ST3, ST4, ST5)
               (ST0, ST1, ST2, ST3, ST4, ST5, ST6)
               (ST0, ST1, ST2, ST3, ST4, ST5, ST6, ST7)
               (ST0, ST1, ST2, ST3, ST4, ST5, ST6, ST7, ST8)
             and 24 others
     = note: required for `table` to implement `LoadQuery<'_, _, model::CostItems>`
note: required by a bound in `diesel::RunQueryDsl::load`
    --> /home/cchee/.cargo/registry/src/github.com-1ecc6299db9ec823/diesel-2.0.3/src/query_dsl/mod.rs:1499:15
     |
1499 |         Self: LoadQuery<'query, Conn, U>,
     |               ^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `diesel::RunQueryDsl::load`

My code can be found here: https://github.com/cchee/rust-tryout

**Comment by weiznich** [comment]
Duplicate of https://github.com/diesel-rs/diesel/discussions/3450 (and any other discussion that shows up if you search for `CompatibleType`).
(As additional note: I spend as much effort on answering questions as people spend effort in asking these question. I consider this question really low effort as it does not even contain an actual question. Feel free to put more information in your post to get a better answer.)
